collections/hrsi-fsc.yaml

```diff
@@ -0,0 +1,82 @@
+Name: High-Resolution Snow & Ice Monitoring; Fractional Snow Cover
+Description: |
+  The High-Resolution Snow & Ice Monitoring service (HR-S&I) is part of the Copernicus Land Monitoring Service (CLMS). The snow aspect of the service provides products 
+  measuring Snow cover (FSC, FSTOC, FSCOG, GFSC), Snow state conditions (WDS, SWS) and persistent snow area (PSA). There are also ice products that measure ice cover 
+  (RLIE) and aggregated river and lake ice extent (ARLIE).
+
+  More information about the data can be found [here](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring) and
+  the snow products user manual is located [here](https://land.copernicus.eu/user-corner/technical-library/hrsi-snow-pum). 
+
+  The Fractional Snow Cover (FSC) product provides the snow fraction at the Top Of Canopy (FSCTOC) and On Ground (FSCOG). NDSI is also provided as part of this service.
+AdditionalInfoExternal:
+    Title: Additional info 
+    Path: hrsi-fsc/README.MD 
+Image: hrsi-fsc/thumbnail.png       
+Resolution: 20m
+GeographicalCoverage: EEA39
+TemporalAvailability: 1st September 2016 - present
```

Comment:
For uniformity, I suggest to keep the current convention

`September 2016 - ongoing` 

same for the rest of  the products